src/components/Calculator.js

```diff
@@ -0,0 +1,95 @@
+/* eslint jsx-quotes: ["error", "prefer-single"] */
```

Comment:
- [OPTIONAL] it is always a good practice to make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉 